modules/va_forms/app/workers/va_forms/form_builder.rb

```diff
@@ -84,15 +84,22 @@ def get_sha256(content)
       end
     end
 
-    def update_sha256(form)
+    def content(url)
       time_initial = Time.zone.now
+      content = URI.parse(url).open
+      Rails.logger.info("Time to open #{form.url}: #{Time.zone.now - time_initial}")
```

Comment:
Could also track this as a [DataDog duration event,](https://github.com/department-of-veterans-affairs/vets-api/blob/50abfe7f58b62efe0d7d9273ed871b8f53b56d06/modules/claims_api/lib/bgs_service/local_bgs.rb#L264) which would give us fancy graphs and alerts from the APM instead of hard-coded, if that sound useful. Not necessary, just pointing it out :)